src/Freya.Core/Operators.fs

```diff
@@ -17,46 +17,79 @@
 // limitations under the License.
 //----------------------------------------------------------------------------
 
-module Freya.Core.Operators
+namespace Freya.Core
 
-(* Operators *)
+/// Core combinator definitions for <see cref="Freya{T}" /> computations.
+[<CompilationRepresentation(CompilationRepresentationFlags.ModuleSuffix)>]
+module Freya =
```

Comment:
Do we want to make this a [<RequireQualifiedAccess>] module? It seems to imply it from the usage...